src/DataStructures/SliceVariables.hpp

```diff
@@ -0,0 +1,109 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <ostream>
+
+#include "DataStructures/Index.hpp"
+#include "DataStructures/SliceIterator.hpp"
+#include "DataStructures/Variables.hpp"
+#include "ErrorHandling/Assert.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+// @{
+/*!
+ * \ingroup DataStructuresGroup
+ * \brief Slices the data within `vars` to a codimension 1 slice. The
+ * slice has a constant logical coordinate in direction `sliced_dim`,
+ * slicing the volume at `fixed_index` in that dimension.  For
+ * example, to get the lower boundary of `sliced_dim`, pass `0` for
+ * `fixed_index`; to get the upper boundary, pass
+ * `extents[sliced_dim] - 1`.
+ *
+ * \see add_slice_to_data
+ *
+ * returns Variables class sliced to a hypersurface.
```

Comment:
`\returns` or capitalize if this in intentional because of return-by-pointer.

Reply:
switched to `Returns`